build/uuid-redirects.json

```diff
@@ -84,6 +85,7 @@
     "Compendium.pf2e.equipment-effects.Item.vOgD9wfStLX1utte": "Compendium.pf2e.equipment-effects.Item.Effect: Skyrider Sword",
     "Compendium.pf2e.equipment-effects.Item.vKooOkXHvtqCgZYg": "Compendium.pf2e.equipment-effects.Item.Effect: Energized Cartridge",
     "Compendium.pf2e.equipment-effects.Item.vOU4Yv2MyAfYBbmF": "Compendium.pf2e.equipment-effects.Item.Effect: Aeon Stone Resonance (Amplifying)",
+    "Compendium.pf2e.equipment-effects.Item.wTZnKkT0K4Tdy8mD": "Compendium.pf2e.equipment-effects.Item.Effect: Bravo's Brew",
```

Comment:
Deleted 3 but only redirecting 2?

Reply:
> Deleted 3 but only redirecting 2?

The "new" one is just the existing lesser one, repurposed